src/style.css

```diff
@@ -0,0 +1,39 @@
+body {
+  margin: 0;
+}
+
+.stars {
+  display: flex;
+}
+
+.stars__star {
+  width: 15px;
+  height: 15px;
+  margin-right: 4px;
+  background-repeat: no-repeat;
+  background-position: center;
+}
+
+.stars > .stars__star:nth-child(n) {
+  background-image: url(images/star.svg);
```

Comment:
add background-image to .stars__star
now this selector (.stars > .stars__star:nth-child(n)) says `every child of stars block`, which is .stars__star